### ISSUE TITLE
Transition between photos is very slow and should be optionnal

### ISSUE THREAD

**Issue by dkm** [cozy/cozy-photos-v2#39]
I find the transition between pictures very intrusive. Also, it is very slow on the few computers I have at hand. I have no problem using other similar photo galleries (picasa, flickr, piwigo with fancy js scripts, gallery2, ...)

It would be good to be able to disable it...

**Comment by frankrousseau** [comment]
Yes, it's very annoying. We would like to change the photo browser for a more minimalistic one.
